playbook/app/pb_kits/playbook/pb_button/_button.scss

```diff
@@ -63,6 +64,24 @@ $pb_button_sizes: (
     color: $text_lt_lighter;
   }
 
+  &.rounded {
+    padding: $space_xs;
+    min-height: unset;
+    background-color: transparent;
+    border-radius: $border_radius_lg;
+    &:hover {
+      color: $primary;
+      background-color: $secondary_action;
+      [class^=pb_detail_kit][class*=color_default] {
+        color: $primary;
+      }
+      &:active {
+        color: $text_lt_default;
+      }
+    }
+  }
```

Comment:
```suggestion
```

Can you move this to a local component for now. We can circle back if this will be made into a kit.

Reply:
@jasoncypret Ready for review.